src/main/java/mate/academy/dao/impl/MovieSessionDaoImpl.java

```diff
@@ -66,7 +67,11 @@ public MovieSession add(MovieSession movieSession) {
     @Override
     public Optional<MovieSession> get(Long id) {
         try (Session session = HibernateUtil.getSessionFactory().openSession()) {
-            return Optional.ofNullable(session.get(MovieSession.class, id));
+            Query<MovieSession> query = session.createQuery("FROM MovieSession ms "
+                    + "LEFT JOIN FETCH ms.movie m LEFT JOIN FETCH ms.cinemaHall ch "
```

Comment:
```suggestion
                    + "JOIN FETCH ms.movie m LEFT JOIN FETCH ms.cinemaHall ch "
```
in this case we can use inner join